### ISSUE TITLE
Feature Request: Allow query parameter

### ISSUE THREAD

**Issue by toru-takahashi** [treasure-data/luigi-td#3]
When I issue query, I want to allow "--priority".
But, currently, there is no way.

**Comment by k24d** [comment]
Now you can set priority like this:

```
class MyQuery(Query):
    type = 'hive'
    priority = -1
    database = 'sample_datasets'
    def query(self):
        return 'select count(1) cnt from www_access'
```

**Comment by k24d** [comment]
Released 0.6.6.
